From 144b28b782e49cba22c24473055c025517013c18 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 22 Jan 2026 19:31:36 +0000
Subject: [PATCH] Fix crash on quit

It's not needed to call `delete this` on quit, and it leads to a
segmentation fault.
---
 floppy.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/floppy.cpp b/floppy.cpp
index f119fa5..46705f5 100644
--- a/floppy.cpp
+++ b/floppy.cpp
@@ -425,7 +425,6 @@ void FloppyData::quit()
         formatActions->quit();
     writeSettings();
     qApp->quit();
-    delete this;
 }
 
 void FloppyData::setEnabled(bool b)
-- 
GitLab

